app/src/main/java/berthier/antoine/birudroid/model/User.kt

```diff
@@ -0,0 +1,17 @@
+package berthier.antoine.birudroid.model
+
+import android.graphics.Bitmap
+import android.media.Image
+import android.util.JsonReader
+import com.google.gson.Gson
+import java.util.LinkedHashSet
+
+class User(
+    var name: String,
+    var mail: String,
+    var id: String,
+    var beerFavorite: ArrayList<Beer> = ArrayList<Beer>(),
+    var beerNote: ArrayList<String> = ArrayList<String>()
```

Comment:
Class note à crée avec un idBeer et un float sinon comment associer la note à la bière ?

Reply:
Il faut pas associer une note a la bière, il faut simplement que le user se souvienne des bières qu’il a notées. Sauf si tu veux aussi qu’il se souvienne de la note pour un éventuel affichage.

Reply:
En tant qu'utilisateur qd tu cliques sur une bière tu aimes savoir cb tu lui as mis :smile:

Reply:
Oki, c'est partie pour une `data class notes`